### PR TITLE
add postgress db; refactor code to respond with data from db

### PR DESCRIPTION
**Pull request by bahkostya**
Used Docker image for running posgres 
___
### To start application:

1. To start db instance run `npm run db:start`
2. To create tables in DB run `npm run db:migrate`
3. To import data to DB run `npm run db:seed`
4. Run app `npm start`

### CODE REVIEW

File: create-connection.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import Sequelize from 'sequelize';
+
+export default new Sequelize(
```
Comment (kulism) [inline create-connection.js]: Seems it's not used anywhere

File: controllers/products.js [hunk 2]
```diff
@@ -2,38 +2,42 @@ import { Product } from '../models';
...
     } catch (error) {
         next(error);
     }
 }
 
-export function getProductById(req, res, next) {
+export async function getProductById(req, res, next) {
     try {
-        res.json(Product.getById(req.params.id));
+        res.json(Product.mapProduct(await Product.getById(req.params.id)));
```
Comment (kulism) [inline controllers/products.js]: In some cases, like here, products are already mapped on the model level, in other like Product.getAll() they need further mapping on the controller level. I would try to stay coherent, and use one apporach. I personally would like the model (not the db one) to return already mapped response. No need to expose the dto to controller.

File: create-connection.js (hunk 1 above)
Comment (bahkostya) [reply create-connection.js]: sure, forgot to delete it after generating config using CLI

File: controllers/products.js (hunk 2 above)
Comment (bahkostya) [reply controllers/products.js]: felt the same, didn't know where to place it better, will move mapping to model

**Pull request merged**